build/docker/Makefile

```diff
@@ -4,17 +4,15 @@ include $(CONFIG_FILE)
 #
 # NOTE: Working directory when calling this file must be project root.
 
-IMG_TAG=fmil:test
-
 .PHONY: docker_image install test documentation docker_run_debug
 
 docker_image:
-	docker build -f build/docker/$(DOCKERFILE_NAME) -t $(IMG_TAG) .
+	docker build -f build/docker/$(DOCKERFILE_NAME) -t $(DOCKER_IMAGE_TAG) .
```

Comment:
Here's the main change. Before the tag name was `fmil:test` for both centos and ubuntu. When the jobs run in parallel on the same machine, this causes a race condition. Now the name will unique for each platform.

Here is still a race condition. Let's say I make a change in the image on my dev branch and start a build at same time as some other job is building. Then they will have the same name but different image content, so same issue. The way to get around it would be to have a separate build to create the image like we do in other projects, but I don't feel it's worth the effort right now. (This in general applies for any Jenkins job which builds a docker image as part of CI testing.)